src/2_banner.js

```diff
@@ -144,11 +144,15 @@ banner = function(branch, options, linkData, storage) {
 				sendSMS(doc, branch, options, linkData);
 			});
 		}
+
 		var closeButton = doc.getElementById('branch-banner-close');
+		banner.close = function() {
```

Comment:
hrmm, seems a little weird to set a new method on banner like this. not sure how to rectify though. somewhere we gotta store a pointer to the element. maybe just return this function, and the branch object can store this function and retrieve it later? or maybe that is not as clean?